internal/test/test.go

```diff
@@ -133,8 +133,17 @@ func importResolver(scriptPath string, state *flowkit.State) cdcTests.ImportReso
 		if !isFileImport {
 			return "", fmt.Errorf("cannot import from %s", location)
 		}
-
 		relativePath := stringLocation.String()
+
+		if strings.Contains(relativePath, "_helper") {
```

Comment:
Maybe strip the file extension (`path/filepath` `Ext()`) and check the plain filename contains this as a suffix. Or is the idea that this allows both `_helper` and e.g. `_helpers` like in the two test files?

Also, maybe refactor this magic string into a constant and add a comment explaining it.

Reply:
> Or is the idea that this allows both _helper and e.g. _helpers like in the two test files?

Yes, exactly :+1: It might be the case that developers will put everything in a single helper file, or split them in several files. So I wanted to allow some flexibility in both plural & singular names.

> Also, maybe refactor this magic string into a constant and add a comment explaining it.

Fixed in https://github.com/onflow/flow-cli/pull/1030/commits/e920d18d486172b5d9dc225ed5a2fb1db92080d8